tools/rccl-prim-test/rccl_prim_test.cpp

```diff
@@ -340,11 +352,14 @@ int main(int argc,char* argv[])
     double speed = (double)(profiling_data_0->bytes_transferred) / (deltaSec*1.0E9);
     printf("Transfered %lu bytes in %f s. Throughput %f GB/s\n", profiling_data_0->bytes_transferred, deltaSec, speed);
 
-    fprintf(stderr, "GPU 0: write_cycles %ld bytes_transferred %ld\n",
-      profiling_data_0->write_cycles, profiling_data_0->bytes_transferred);
+#define RTC_CLOCK_FREQ 2.7E07
```

Comment:
Where did this number come from?

Reply:
From Sean. Unfortunately I can't find the number from datasheet